pkg/skaffold/util/cmd.go

```diff
@@ -74,7 +74,7 @@ func (*Commander) RunCmdOut(cmd *exec.Cmd) ([]byte, error) {
 
 	err = cmd.Wait()
 	if err != nil {
-		return stdout, errors.Wrapf(err, "Running %s: stdout %s, stderr: %s, err: %v", cmd.Args, stdout, stderr, err)
+		return stdout, errors.Wrapf(err, "Running %s\n - stdout: %s\n - stderr: %s", cmd.Args, stdout, stderr)
```

Comment:
Embedding newlines doesn't seem great.  It feels like this should be an object.

Reply:
We could do that. And it would have an Error() function that embeds newlines...
I'd rather keep this simple form

Reply:
And that object would have an Error() function with that string... I'd rather keep this simple form